crates/zome_types/src/entry.rs

```diff
@@ -27,9 +27,64 @@ pub type CapGrantEntry = ZomeCallCapGrant;
 /// The data type written to the source chain to denote a capability claim
 pub type CapClaimEntry = CapClaim;
 
-#[derive(PartialEq, Debug, Default, Clone, Serialize, Deserialize)]
-/// @todo make some options for get
-pub struct GetOptions;
+#[derive(PartialEq, Debug, Clone, Serialize, Deserialize)]
+/// Options for controlling how get works
+pub struct GetOptions {
+    /// If this is true the get call will wait for
+    /// the latest data before returning.
+    /// If it is false you will get whatever is locally
+    /// available on this conductor.
+    pub call: GetCall,
```

Comment:
Isn't `GetOptions` already about the `get` call? Seems confusing that there's an inner layer for the get call options. Is "call" the right name for this aspect of `get`? What about `strategy`?

```suggestion
    pub strategy: GetStrategy,
```